defaults/main.yml

```diff
@@ -5,6 +5,8 @@ grub_consoles:
   - tty0
   - 'ttyS0,{{ grub_serial.speed }}'
 
+grub_serial_enabled: True
+
 # grub serial command settings
 grub_serial:
```

Comment:
I have actually tried this, the issue was that if you have a system with no serial console you'd actually have to override "enabled" + ALL the other elements needlessly (speed, unit, word, parity and stop) since if you only define i.e.

```
grub_serial:
  enabled: False
```
It will fail because you end up with the remaining element being undefined which trips Ansible.